docs/csharp/nullable-references.md

```diff
@@ -55,25 +96,26 @@ The compiler uses the following rules in a disabled nullable context:
 
 The behavior is the same as previous versions of C#.
 
-The compiler uses the following rules in an enabled nullable context:
+The compiler uses the following rules in an enabled nullable annotation context:
 
 - Any variable of a reference type is a **non-nullable reference**.
 - Any non-nullable reference may be dereferenced safely.
 - Any nullable reference type (noted by `?` after the type in the variable declaration) may be null. Static analysis determines if the value is known to be non-null when it is dereferenced. If not, the compiler warns you.
 - You can use the null-forgiving operator to declare that a nullable reference isn't null.
 
-A richer grammar is proposed for nullable contexts and will be available in future previews. For more information about nullable contexts, see the [nullable reference specification](https://github.com/dotnet/csharplang/blob/master/proposals/nullable-reference-types-specification.md#nullable-contexts) draft.
+In an enabled nullable annotation context, the `?` character appended to a reference type declares a **nullable reference type**. The **null forgiveness operator** (`!`) may be appended to an expression to declare that the expression is not null.
```

Comment:
Does the `!` operator have an expression type which is always non-null as well, or does it only affect the static analysis?

Reply:
When I read the [spec](https://github.com/dotnet/csharplang/blob/master/proposals/nullable-reference-types-specification.md#the-null-forgiving-operator) I think it does not change the expression type from `nullable_reference_type` to `nonnullable_reference_type`. It only declares that the variable is currently "not null".

  /cc @MadsTorgersen for verification.